validmind/tests/output.py

```diff
@@ -43,15 +43,9 @@ def process(self, item: Any, result: TestResult) -> None:
         result.passed = bool(item)
 
 
-class MetricOutputHandler(OutputHandler):
+class MetricValuesOutputHandler(OutputHandler):
     def can_handle(self, item: Any) -> bool:
-        # Accept individual numbers
-        if isinstance(item, (int, float)):
-            return True
-        # Accept lists/arrays of numbers for per-row metrics
-        if isinstance(item, (list, tuple, np.ndarray)):
-            return all(isinstance(x, (int, float, np.number)) for x in item)
-        return False
+        return isinstance(item, (int, float, list))
```

Comment:
Does this mean we can still save a list of values for a unit metric? Since the backend can't save that.